app/modules/ads.js

```diff
@@ -364,6 +364,10 @@ class Ads {
 		return !!document.querySelector('.wds-global-footer');
 	}
 
+	isArticleSectionCollapsed() {
+		return this.adsContext && this.adsContext.opts && this.adsContext.opts.mobileSectionsCollapse;
```

Comment:
You can reach any variable in adsContext using `get`, `this.adsContext('opts.mobileSectionsCollapse')` should work